src/transformers/generation/utils.py

```diff
@@ -1192,14 +1193,18 @@ def generate(
                 on the batch ID `batch_id` and the previously generated tokens `inputs_ids`. This argument is useful
                 for constrained generation conditioned on the prefix, as described in [Autoregressive Entity
                 Retrieval](https://arxiv.org/abs/2010.00904).
-            synced_gpus (`bool`, *optional*):
+            synced_gpus (`bool`, *optional*, defaults to `False`):
```

Comment:
This should be left as is.

Reply:
oops, probably some unwanted rebase change! 🙈